browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -1621,6 +1621,9 @@ Are you sure you want to do this?
       <message name="IDS_REPORT_BROKEN_SITE_TEXT_2" desc="Shields text for broken text warning message.">
         Note: This site address will be submitted with your Brave version number and your IP address(which will not be stored).
       </message>
+      <message name="IDS_REPORT_ERROR_PAGE_EXPLANATION" desc="Notifies the user that the error page cannot be reported">
+        This page is an error page and is not eligible for reporting.
```

Comment:
Maybe simplify a bit: "This is an error page. It is not eligible for reporting." cc: @rmcfadden3

Reply:
Maybe like `This is an error page. It cannot be reported as a broken site.`